exonum-java-binding/core/rust/ejb-app/TUTORIAL.md

```diff
@@ -97,11 +109,12 @@ $ cargo run -- run -d testnet/db -c testnet/node.toml --public-api-address 127.0
     --ejb-port 6000
 ```
 
-There are few specific optional parameters here:
+There are few optional parameters here:
 - `--jvm-args-prepend` and `--jvm-args-append`: Additional parameters for JVM that prepend and
  append the rest of arguments. Must not have a leading dash. For example, `Xmx2G`.
 - `--jvm-debug`: Allows JVM being remotely debugged over the `JDWP` protocol. Takes a socket address as a parameter in form
  of `HOSTNAME:PORT`. For example, `localhost:8000`.
+- `--ejb-log-config-path`: Path to log4j configuration file.
```

Comment:
What should be used as a default behavior? Disabled logging or console output?

Reply:
Console output AFAIR